node/follower_node_test.go

```diff
@@ -169,3 +172,137 @@ func TestFastCatchupResume(t *testing.T) {
 	// Verify the sync was reset.
 	assert.Equal(t, uint64(0), node.GetSyncRound())
 }
+
+// TestDefaultResourcePaths confirms that when no extra configuration is provided, all resources are created in the dataDir
+func TestDefaultResourcePaths_Follower(t *testing.T) {
```

Comment:
is it possible to consolidate (some of) the follower/full tests? cc: @Eric-Warehime